include/revng/Model/model-schema.yml

```diff
@@ -1859,3 +1874,31 @@ definitions:
         doc: |-
           Print all the addresses exactly how disassembler emitted them
           in PC-relative mode.
+
+  - name: BinaryIdentifier
```

Comment:
1. I'm OK with this for now, but please backlog renaming `model::Binary` to `model::Project` and turning `BinaryIdentifier` into just `Binary`. It's time to do this.
2. Move this before the definition of segment. Let's strive to keep things in some "reading order".